src/components/GEM2SLoadingScreen.jsx

```diff
@@ -106,7 +105,7 @@ GEM2SLoadingScreen.propTypes = {
 
 GEM2SLoadingScreen.defaultProps = {
   completedSteps: [],
-  steps: [],
+  steps: null,
```

Comment:
looks like you are not passing steps in props at all, in which case it is better to remove it alltogether.
Also, why do we not pass it anymore? Are any other components that call `GEM2SLoadingScreen` passing it?

Reply:
@ivababukova The `steps` prop is only used for testing. Do you think it is a valid use case, or should I kill it and have the real steps hardcoded inside the component rather than having them as the default props? See https://github.com/biomage-ltd/ui/pull/274/commits/e909c5f8b83ad6691432991b96d60bfe8ca13077

Reply:
I removed and adapted the tests